lib/src/themes/paint_factory.dart

```diff
@@ -28,21 +28,27 @@ class PaintExpression extends Expression<Paint> {
       };
 }
 
+class VectorPaint extends Paint {
```

Comment:
There's nothing specific to vectors about his class. Let's name it for what it is, e.g. `PaintModel`